src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -1741,6 +1740,44 @@ function getSavedObjectNamespaces(
   return [SavedObjectsUtils.namespaceIdToString(namespace)];
 }
 
+/**
+ * Gets a saved object from a raw OpenSearch document.
+ *
+ * @param registry
+ * @param type
+ * @param id
+ * @param doc
+ *
+ * @internal
+ */
```

Comment:
If we're going to add params can we add a description for each?

Reply:
good catch. we should + added.